modules/imgproc/perf/opencl/perf_imgproc.cpp

```diff
@@ -47,6 +47,8 @@
 #include "../perf_precomp.hpp"
 #include "opencv2/ts/ocl_perf.hpp"
 
+#ifdef HAVE_OPENCL
```

Comment:
Actually there is a plan to drop whole file `#ifdef HAVE_OPENCL` guards from performance tests.

These tests are misnamed. They copied from 2.4 branch "as is" with `OCL_` prefix to keep test names the same (to compare results between branches).

Correct naming for these tests should have `TAPI_` prefix (see OpenCV T-API announcement from 3.0 release). Tests may run on both CPU and OpenCL code paths (depending on external configuration / H/W availability)